docs/wip/troubleshoot_prometheus.md

```diff
@@ -0,0 +1,118 @@
+---
```

Comment:
If this is ready, this can be dropped under `docs/content/docs/troubleshooting/observability`

Reply:
Yeah I think this is enough to publish. Will move this there.